app/views/providers/show.html.erb

```diff
@@ -41,9 +41,6 @@
   </div>
   <aside class="govuk-grid-column-one-third">
     <div class="app-related">
-      <% if @provider_view.show_notifications_link? %>
```

Comment:
I'm not sure if we use `@provider_view.show_notifications_link` anywhere else but could we remove https://github.com/DFE-Digital/publish-teacher-training/blob/master/app/view_objects/provider_view.rb#L7-L9 ?